apps/console/src/features/authentication-flow-builder/components/authentication-flow-visual-editor.tsx

```diff
@@ -515,10 +520,13 @@ const AuthenticationFlowVisualEditor: FunctionComponent<AuthenticationFlowVisual
                 : AuthenticationSequenceType.USER_DEFINED
         };
 
-        if (!isAdaptiveAuthAvailable
-                || !isConditionalAuthenticationEnabled
-                || AdaptiveScriptUtils.isEmptyScript(authenticationSequence.script)) {
-            sequence.script = AdaptiveScriptUtils.generateScript(authenticationSequence?.steps?.length + 1).join("\n");
+        if (
+            !isAdaptiveAuthAvailable
+            || !isConditionalAuthenticationEnabled
+            || AdaptiveScriptUtils.isEmptyScript(authenticationSequence.script)
+            || orgType === OrganizationType.SUBORGANIZATION
+        ) {
+            sequence.script = "";
```

Comment:
We need to set the script at the root organization level

`sequence.script = AdaptiveScriptUtils.generateScript(authenticationSequence?.steps?.length + 1).join("\n");`

Reply:
Fixed with f35e969